client/src/components/WizardFive/WizardFive.js

```diff
@@ -43,7 +64,7 @@ class WizardFive extends Component {
                     </Link>
 
                     <Link to={'/dashboard'}>
-                    <button className="completeButton">Complete</button>
+                    <button className="completeButton" onClick={() => this.addProperty}>Complete</button>
```

Comment:
doesn't `this.addProperty` need to be invoked?

Reply:
Yep :) I'll fix that.

Reply:
:+1: